packages/sdk/test/evm/signature-drop-v4.test.ts

```diff
@@ -39,7 +39,7 @@ describe("Signature drop tests (v4)", async () => {
   beforeEach(async () => {
     sdk.updateSignerOrProvider(adminWallet);
 
-    signatureDropContract = await sdk.getSignatureDrop(
+    signatureDropContract = await sdk.getContract(
```

Comment:
minor update due to deprecation, this seems to be the last place the [getSignatureDrop](https://github.com/alexmylonas/third-web-js/blob/main/packages/sdk/src/evm/core/sdk.ts#L216) is used internally.

Reply:
Good call, updated!